django/core/validators.py

```diff
@@ -567,9 +567,9 @@ class FileExtensionValidator:
 
     def __init__(self, allowed_extensions=None, message=None, code=None):
         if allowed_extensions is not None:
-            allowed_extensions = [
-                allowed_extension.lower() for allowed_extension in allowed_extensions
-            ]
+            allowed_extensions = sorted(
+                [allowed_extension.lower() for allowed_extension in allowed_extensions]
+            )
```

Comment:
I'd move `sorted()` calls to the `__eq__()`.

Reply:
> I'd move `sorted()` calls to the `__eq__()`.

Fixed it.